tox.ini

```diff
@@ -178,23 +177,26 @@ commands =
 [testenv:eco]
 description = Smoketest of combining all known to be maintained plugins (ecosystem)
 pip_pre = true
-extras =
-    # we install test extra in order to validate it does not drag ansible in
-    test
 deps =
+    # we install test extra in order to validate it does not drag ansible in
```

Comment:
In the future, it's best to move those dependencies to a proper location because the end-users don't need it (plus it's easier to maintain and repin automatically).